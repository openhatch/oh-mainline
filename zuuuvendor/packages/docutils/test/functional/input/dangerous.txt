Potentially dangerous features (security holes):

.. include:: /etc/passwd
.. raw:: html
   :file: /etc/passwd
.. raw:: html
   :url: file:///etc/passwd
.. raw:: html

   <script>
       that does something really nasty
   </script>
.. csv-table:: :file: /etc/passwd
.. csv-table:: :url: file:///etc/passwd
.. figure:: picture.png
   :figwidth: image
