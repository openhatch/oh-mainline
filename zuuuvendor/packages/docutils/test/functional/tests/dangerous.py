# Source and destination file names.
test_source = "dangerous.txt"
test_destination = "dangerous.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html"

# Settings
settings_overrides['file_insertion_enabled'] = False
settings_overrides['raw_enabled'] = False
# local copy of default stylesheet:
settings_overrides['stylesheet_path'] = ( 
    'functional/input/data/html4css1.css')
