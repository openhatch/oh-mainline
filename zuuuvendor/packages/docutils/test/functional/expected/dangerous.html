<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
<title></title>
<link rel="stylesheet" href="../input/data/html4css1.css" type="text/css" />
</head>
<body>
<div class="document">


<p>Potentially dangerous features (security holes):</p>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 3)</p>
<p>&quot;include&quot; directive disabled.</p>
<pre class="literal-block">
.. include:: /etc/passwd
</pre>
</div>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 4)</p>
<p>&quot;raw&quot; directive disabled.</p>
<pre class="literal-block">
.. raw:: html
   :file: /etc/passwd
</pre>
</div>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 6)</p>
<p>&quot;raw&quot; directive disabled.</p>
<pre class="literal-block">
.. raw:: html
   :url: file:///etc/passwd
</pre>
</div>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 8)</p>
<p>&quot;raw&quot; directive disabled.</p>
<pre class="literal-block">
.. raw:: html

   &lt;script&gt;
       that does something really nasty
   &lt;/script&gt;
</pre>
</div>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 13)</p>
<p>File and URL access deactivated; ignoring &quot;csv-table&quot; directive.</p>
<pre class="literal-block">
.. csv-table:: :file: /etc/passwd
</pre>
</div>
<div class="system-message">
<p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">functional/input/dangerous.txt</tt>, line 14)</p>
<p>File and URL access deactivated; ignoring &quot;csv-table&quot; directive.</p>
<pre class="literal-block">
.. csv-table:: :url: file:///etc/passwd
</pre>
</div>
<div class="figure">
<img alt="picture.png" src="picture.png" />
</div>
</div>
</body>
</html>
